src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,116 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int CAPACITY = 16;
+    private static final int MAX_CAPACITY = Integer.MAX_VALUE / 2;
```

Comment:
i think we don't need MAX_CAPACITY in our primitive realization
```suggestion
    private static final int DEFAULT_CAPACITY = 16;
```